<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>粉專即時推播器 @MUSE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QPlainTextEdit" name="post_plainText_edit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>341</width>
        <height>401</height>
       </rect>
      </property>
      <property name="centerOnScroll">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>請輸入欲發送之貼文內容</string>
      </property>
     </widget>
     <widget class="QLabel" name="file_path_label">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>檔案路徑：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="image_path_lineEdit">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="upload_image_button">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>選擇圖片</string>
      </property>
     </widget>
     <widget class="QLabel" name="image_preview_label">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>381</width>
        <height>401</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border :1.5px black;
border-style : dashed;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>預覽圖片(可拖曳至此)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="execution_log_TextEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>480</y>
        <width>581</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cascadia Mono</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: black; color: white;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="undoRedoEnabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>執行紀錄：</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>480</y>
        <width>160</width>
        <height>83</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_1">
       <item>
        <widget class="QPushButton" name="reset_button">
         <property name="text">
          <string>恢復預設</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="post_setting_button">
         <property name="text">
          <string>發文對象</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="publish_post_button">
         <property name="text">
          <string>發布貼文</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>設置</string>
     </attribute>
     <widget class="QLabel" name="access_token_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>USER ACCESS TOKEN：</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="token_textEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>261</width>
        <height>121</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="switch_state_label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>53</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ON</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="warning_textEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>160</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgba(255, 0, 0, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft JhengHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;短期 ACCESS_TOKEN 有時效性，&lt;br /&gt;或是更換長期 ACCESS_TOKEN，&lt;br /&gt;保存於本地端有洩漏ACCESS_TOKEN的風險。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="long_live_token_button">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>130</y>
        <width>251</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>產生長期 ACCESS TOKEN(Comming Soon)</string>
      </property>
     </widget>
     <widget class="QLabel" name="api_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>API Version：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="api_LineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="switch_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>53</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Switch</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
